yup/package.json

```diff
@@ -11,6 +11,7 @@
   "types": "dist/index.d.ts",
   "license": "MIT",
   "peerDependencies": {
-    "react-hook-form": ">=6.6.0"
+    "react-hook-form": ">=6.6.0",
```

Comment:
FYI:I think with the latest update (i will move to 7) there are breaking changes.